torchtune/training/checkpointing/_utils.py

```diff
@@ -460,6 +460,9 @@ def get_adapter_checkpoint_path(
 
     Returns:
         Optional[Path]: Path to the adapter checkpoint file, or None if not applicable.
+
+    Raises:
+        ValueError: If the adapter checkpoint file is missing or If the adapter checkpoint file is not a .pt file.
```

Comment:
```suggestion
        ValueError: If the adapter checkpoint file is missing or if the adapter checkpoint file is not a .pt file.
```
Out of curiosity, when would adapter checkpoint not be  a .pt file?

Reply:
we save both .safetensors (for HF) and .pt. Its ugly, but we dont have a function to load ckpt as .safetensors.